process-engine/bpmn-studio
# 💄 Move Toastr Notifications

## What did you change?

This PR moves the Toastr notifications to the bottom left corner.

Fixes #428 

## How can others test the changes?

- Open the BPMN-Studio

- Go to any Detail View
- Click on the Save-Button
(Or trigger any other notification)

- Notice that the notification is in the bottom left corner

## PR-Checklist

Please check the boxes in this list after submitting your PR:
